content/src/logic/database-queries/snapshots-queries.ts

```diff
@@ -116,6 +116,14 @@ export async function saveSnapshot(
   await database.queryWithValues(query, 'save_snapshot')
 }
 
+export async function hasSnapshot(components: Pick<AppComponents, 'database'>, snapshotHash: string): Promise<boolean> {
```

Comment:
shouldnt this be something like `isLocalSnapshot`?

Reply:
own snapshot

Reply:
own snapshot sound good